parsers/keyval/timeformat.go

```diff
@@ -0,0 +1,66 @@
+package keyval
```

Comment:
yeah...  and I think more likely is to strip it out of keyval entirely - whereas with a JSON line the timefield is almost guaranteed to be buried in a field somewhere, I'm guessing that a keyval line is more likely to have it in the prefix, and we can easily make stronger requirements about the field name (if not format) for that.